reana_workflow_engine_serial/tasks.py

```diff
@@ -44,35 +48,145 @@ def get_job_status(job_id):
     return response
 
 
+def declare_job_status_queue():
+    broker_credentials = pika.PlainCredentials(BROKER_USER,
+                                               BROKER_PASS)
+    connection = pika.BlockingConnection(
+        pika.ConnectionParameters(BROKER_URL,
+                                  BROKER_PORT,
+                                  '/',
+                                  broker_credentials))
+    channel = connection.channel()
+    channel.queue_declare(queue='jobs-status')
+    channel.queue_declare(queue='jobs-progress')
+    return channel
+
+
+def publish_workflow_status(channel, workflow_uuid, status, message=None):
+    """Update database workflow status.
+
+    :param workflow_uuid: UUID which represents the workflow.
+    :param status: String that represents the analysis status.
+    :param status_message: String that represents the message related with the
+       status, if there is any.
+    """
+    log.info('Publishing Workflow: {0} Status: {1}'.format(workflow_uuid,
+                                                           status))
+    channel.basic_publish(exchange='',
+                          routing_key='jobs-status',
+                          body=json.dumps({"workflow_uuid": workflow_uuid,
+                                           "status": status,
+                                           "message": message}),
+                          properties=pika.BasicProperties(
+                              delivery_mode=2,  # msg persistent
+                          ))
+
+
+def publish_workflow_job_progress(channel, workflow_uuid, job_info):
```

Comment:
Do we use `job-progress` anywhere? I see only jobs-status used https://github.com/reanahub/reana-workflow-controller/pull/82/files#diff-d95dcaa8d671d2a3c798faa7fd757a3fR81 and no call to `publish_workflow_job_progress`.

Reply:
Nope, it's a remnant from a previous implementation, removing